admin/setup.cfg

```diff
@@ -7,11 +7,11 @@
 name = securedrop-admin
 version = 0.1.0
 summary = SecureDrop Admin Toolkit
-description-file =
+description_file =
     README.rst
 author = Loic Dachary
```

Comment:
If we're updating this, should we change it to securedrop@freedom.press?

Reply:
Disinclined to change attributions without checking with original authors. But we probably could.